### PR TITLE
feat(Swap&StableSwap&Limit): Add QuickInputButton

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: src/components/CurrencyInputPanel/index.tsx [hunk 1]
```diff
@@ -235,9 +239,26 @@ export default function CurrencyInputPanel({
               </Text>
             )}
             {account && currency && !disabled && showMaxButton && label !== 'To' && (
-              <Button onClick={onMax} scale="xs" variant="secondary" style={{ textTransform: 'uppercase' }}>
-                {t('Max')}
-              </Button>
+              <Flex alignItems="right" justifyContent="right">
+                {showQuickInputButton &&
```
Comment (chef-jojo) [inline src/components/CurrencyInputPanel/index.tsx:243]: can move out the `showMaxButton` check
we can show this component when user has balance

File: src/views/Swap/StableSwap/components/StableSwapForm.tsx [hunk 2]
```diff
@@ -184,6 +184,15 @@ export default function StableSwapForm({ setIsChartDisplayed, isChartDisplayed }
     [onCurrencySelection],
   )
 
+  const handlePercentInput = useCallback(
+    (percent) => {
+      if (maxAmountInput) {
+        onUserInput(Field.INPUT, (Number(maxAmountInput.toExact()) * percent * 0.01).toString())
```
Comment (chef-jojo) [inline src/views/Swap/StableSwap/components/StableSwapForm.tsx:190]: `onUserInput(Field.INPUT, maxAmountInput.multiply(new Percent(percent, 100)).toExact())`

File: src/components/CurrencyInputPanel/index.tsx (hunk 1 above)
Comment (chef-jojo) [reply src/components/CurrencyInputPanel/index.tsx:243]: `selectedCurrencyBalance?.greaterThan(0)`

File: src/views/LimitOrders/index.tsx [hunk 3]
```diff
@@ -159,6 +158,14 @@ const LimitOrders = () => {
     },
     [handleInput],
   )
+  const handlePercentInput = useCallback(
+    (percent) => {
+      if (maxAmountInput) {
+        handleInput(Field.INPUT, (Number(maxAmountInput.toExact()) * percent * 0.01).toString())
```
Comment (chef-jojo) [inline src/views/LimitOrders/index.tsx:164]: update this too

Comment (Woody810) [reply src/views/LimitOrders/index.tsx]: done.

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**